string ptu_cmd_string
